src/content/code/services/apisix.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Apache APISIX
+description: Apache APISIX is a dynamic, real-time, high-performance API gateway. APISIX provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, etc. As a cloud-native API gateway, Apache APISIX already can recognize GraphQL syntax at the beginning of its design. Efficiently matching GraphQL statements carried in requests can filter out abnormal traffic to further ensure security and improve system performance. For more information, go to https://apisix.apache.org/blog/2022/03/02/
```

Comment:
The ideal description length is 70-155 characters. This is just for discoverability and SEO. So, we can make it much smaller.

Reply:
Sure~
Let me try

Reply:
Structurally this all looks good - just let me know when you're happy

Reply:
@orta I think I'm ready to go. Please review, sir.
Thanks first.